tests/ml_commons/test_model_access_control.py

```diff
@@ -11,8 +11,13 @@
 from opensearch_py_ml.ml_commons.model_access_control import ModelAccessControl
 from tests import OPENSEARCH_TEST_CLIENT
 from opensearchpy.exceptions import RequestError
+from packaging.version import parse as parse_version
+import os
+
+OPENSEARCH_VERSION = parse_version(os.environ['OPENSEARCH_VERSION'])
+MAC_MIN_VERSION = parse_version("2.8.0")
+MAC_UPDATE_MIN_VERSION = parse_version("2.11.0")
```

Comment:
One thought: Eventually we will be interested to upgrade the Opensearch version in the test [here](https://github.com/opensearch-project/opensearch-py-ml/blob/main/.github/workflows/integration.yml#L15)

I'm just wondering if we should bump the version there and check what integration tests are failing and may be address those changes?

I understand bumping version is not scope of this issue, but i think it will address multiple issues at the same time.

What do you think?

Reply:
Yes, I think along with having separate runs per versions.. i will note down failing tests to fix